src/components/search/SearchProgramCard.jsx

```diff
@@ -76,7 +78,22 @@ const SearchProgramCard = ({ hit, isLoading, ...rest }) => {
     if (!numCourses) {
       return undefined;
     }
-    return `${numCourses} ${numCourses > 1 ? 'Courses' : 'Course'}`;
+    return `${numCourses} ${numCourses > 1
+      ? (
+        intl.formatMessage({
+          id: 'enterprise.search.page.program.card.footer.for.more.than.one.course',
+          defaultMessage: 'Courses',
+          description: 'Label for pragram card footer when there are more than one course in the program.',
+        })
+      )
+      : (
+        intl.formatMessage({
+          id: 'enterprise.search.page.program.card.footer.for.one.course',
+          defaultMessage: 'Course',
+          description: 'Label for pragram card footer when there is only one course in the program.',
+        })
```

Comment:
It would be great if we can combine these two `formatMessage` into a single `formatMessage` using singular and plural